js/mod.ts

```diff
@@ -54,34 +54,51 @@ export interface BundleOptions {
   allowRemote?: boolean;
   /** The cache root to use, overriding the default inferred `DENO_DIR`. */
   cacheRoot?: string;
+  /** Base URL to resolve import map specifiers. Defaults to `Deno.cwd()`.
+   * Ignored if `imports` is not set. */
+  baseUrl?: URL | string;
   /** The setting to use when loading sources from the Deno cache. */
   cacheSetting?: CacheSetting;
   /** Compiler options which can be set when bundling. */
   compilerOptions?: CompilerOptions;
-  imports?: Record<string, string[]>;
+  /** Location of an import map */
+  importMap?: URL | string;
```

Comment:
An alternative here would be to accept an inline import map as on object on this property then remove the other properties at the top level. For example:

```ts
importMap?: URL | ImportMap | string;
```

Reply:
OK, let's do that. Is it something that will eventually be done for Deno's config?